resources/quality/nwa3d_a5/nwa3d_a5_best.inst.cfg

```diff
@@ -12,118 +12,120 @@ global_quality = True
 
 [values]
 layer_height = 0.08
-initial_layer_height = 0.25
+layer_height_0 = 0.24
 line_width = 0.4
-initial_layer_line_width = 100
+wall_line_width_0 = 100
```

Comment:
This `wall_line_width_0` is not the same as the `initial_layer_line_width`. `wall_line_width_0` is the width of the outer wall. If you want to set the Initial layer line width you have to use `initial_layer_line_width_factor` instead.